website/docs/reference/widgets/filepicker.md

```diff
@@ -1,6 +1,6 @@
 # Filepicker
 
-This document guides you on how to use the Filepicker widget for selecting and uploading various file types, including images, videos, audio, text files, and more.
+This document explains how to use a datapicker widget to capture date and time input from users.
```

Comment:
why is this here?

Reply:
Wait a min, I have no idea :(  I'll fix this